angular.js

```diff
@@ -28,6 +28,8 @@ module.exports = {
     ],
     '@angular-eslint/no-output-native': 'error',
     '@angular-eslint/use-lifecycle-interface': 'error',
+    eqeqeq: 'error',
```

Comment:
Just to confirm, is this supposed to be here?

Reply:
I suppose it's not needed